packages/hash/playwright/tests/page-creation.spec.ts

```diff
@@ -163,41 +158,39 @@ test("user can create page", async ({ page }) => {
 });
 
 // TODO: investigate flakiness of page renaming and enable the test in CI
-test.skip("user can rename page", async ({ page }) => {
-  const changedPageName = `Renamed test page ${pageNameSuffix}`;
+test("user can rename page", async ({ page }) => {
```

Comment:
Hmm another test is now [failing](https://github.com/hashintel/hash/runs/8135546216?check_suite_focus=true) with this message: `User with id .* has created too many verification codes recently`

It must be to do with us running too many tests without resetting the DB. We could get away with this earlier because we were under the limit of how many subsequent logins our backend allowed. Removing `test.skip` added more login attempts to deal with, so we exceeded the limit.

I [relaunched the tests](https://github.com/hashintel/hash/runs/8136317460?check_suite_focus=true) to confirm. If they fail again, we'll need to delete past login attempts from the DB, similarly to have we reset the DB in Block Protocol repo.

Reply:
I guess it's time to implement `resetDb` before tests 😃